apps/main/src/dao/components/PageAnalytics/index.tsx

```diff
@@ -35,12 +34,14 @@ const Analytics = () => {
       <Box flex flexColumn fillWidth flexGap={'var(--spacing-3)'}>
         <CrvStats />
         <Box>
-          <SubNav
-            activeKey={navSelection}
-            navItems={navItems}
-            setNavChange={setNavSelection as (key: string) => void}
-            nested
+          <TabsSwitcher
+            variant="contained"
+            size="medium"
+            value={navSelection}
+            onChange={setNavSelection as (key: string) => void}
```

Comment:
TabSwitcher is generic so this shouldn't be needed

```suggestion
            onChange={setNavSelection}
```